templates/server.xml.epp

```diff
@@ -52,7 +52,7 @@
 <% } -%>
 <% if $jira::proxy { -%>
 <%   jira::sort_hash($jira::proxy).each |$key, $value| { -%>
-                   <%= $key %> = '<%= $value %>'
+                   <%= $key %>="<%= $value %>"
```

Comment:
Can we move this to separate PR so as to show the end user it changed through our Changelog?

Reply:
Can do!  I'll create a separate PR for this.